apps/src/lab2/levelEditors/panels/EditPanels.tsx

```diff
@@ -1,6 +1,7 @@
 import classNames from 'classnames';
 import React, {useCallback, useEffect, useRef, useState} from 'react';
 
+import Checkbox from '@cdo/apps/componentLibrary/checkbox/Checkbox';
```

Comment:
super nit, non blocking: can also be imported like this -  ` import Checkbox from '@cdo/apps/componentLibrary/checkbox';`